proposals/infra/INF-0005-Long-Vector-ExecutionTest-Plan.md

```diff
@@ -0,0 +1,339 @@
+| log2      | Log | |
+| mad       | IMad | |
+| max       | IMax | |
+| min       | IMin | |
+| pow       | [Log, Exp] | [FMul] , [FDiv] ||
+| rcp       | | FDiv | |
+| round     | Round_ne ||
+| rsqrt     | Rsqrt | |
+| sign      | | ZExt, Sub, [ICmpSLT], [FCmpOLT] | |
+| smoothstep| Saturate | FMul, FSub, FDiv ||
+| sqrt      | Sqrt | |
+| step      | | FCmpOLT, Select ||
+| trunc     | Round_z | |
+
+## Float Ops
+
+| Intrinsic | DXIL OPCode | LLVM Instruction | Notes |
+|-----------|--------------|----------|-----------|
+| f16tof32  | LegacyF16ToF32 | |
+| f32tof16  | LegacyF32ToF16 | |
+| isfinite  | IsFinite | |
+| isinf     | IsInf | |
+| isnan     | IsNan | |
+| modf      | Round_z | FSub, Store | |
+
+## Bitwise Ops
+| Intrinsic | DXIL OPCode | LLVM Instruction | Notes |
+|-----------|--------------|----------|-----------|
+| saturate  | Saturate | |
+| reversebits| Bfrev | |
+| countbits | Countbits | |
+| firstbithigh| FirstbitSHi | |
+| firstbitlow| FirstbitLo | |
+
+## Logic Ops
+
+| Intrinsic | DXIL OPCode | LLVM Instruction | Notes |
+|-----------|--------------|----------|-----------|
+| and       | | And, [ExtractElement, InsertElement] | |
+| or        | | Or, [ExtractElement, InsertElement] | |
+| select    | | Select, [ExtractElement, InsertElement] | |
+
+## Reductions
+
+| Intrinsic | DXIL OPCode | LLVM Instruction | Notes |
+|-----------|--------------|----------|-----------|
+| all       | | [FCmpUNE], [ICmpNE] ,
+||| [ExtractElement, And] |
+| any       | | [FCmpUNE], [ICmpNE] ,
+||| [ExtractElement, Or] | |
+| clamp     | [UMax, UMin], [IMax, IMin] | |
+| dot       | | ExtractElement, Mul | |
+
+
+## Derivative and Quad Operations
+
+| Intrinsic | DXIL OPCode | LLVM Instruction | Notes |
+|-----------|--------------|----------|-----------|
+| ddx       | DerivCoarseX | |
+| ddx_coarse| DerivCoarseX | |
+| ddx_fine  | DerivFineX | |
+| ddy       | DerivCoarseY | |
+| ddy_coarse| DerivCoarseY | |
+| ddy_fine  | DerivFineY | |
+| fwidth    | QuadReadLaneAt | |
+| QuadReadLaneAcrossX | QuadOp | |
+| QuadReadLaneAcrossY | QuadOp | |
+| QuadReadLaneAcrossDiagonal | QuadOp | |
+
+## WaveOps
+
+| Intrinsic | DXIL OPCode | LLVM Instruction | Notes |
+|-----------|--------------|----------|-----------|
+| WaveActiveBitAnd | WaveActiveBit | |
+| WaveActiveBitOr  | WaveActiveBit | |
+| WaveActiveBitXor | WaveActiveBit | |
+| WaveActiveProduct| WaveActiveOp | |
+| WaveActiveSum    | WaveActiveOp | |
+| WaveActiveMin    | WaveActiveOp | |
+| WaveActiveMax    | WaveActiveOp | |
+| WaveMultiPrefixBitAnd | WaveMultiPrefixOp | |
+| WaveMultiPrefixBitOr  | WaveMultiPrefixOp | |
+| WaveMultiPrefixBitXor | WaveMultiPrefixOp | |
+| WaveMultiPrefixProduct| WaveMultiPrefixOp | |
+| WaveMultiPrefixSum    | WaveMultiPrefixOp | |
+| WavePrefixSum         | WavePrefixOp | |
+| WavePrefixProduct     | WavePrefixOp | |
+| WaveReadLaneAt        | WaveReadLaneAt | |
+| WaveReadLaneFirst     | WaveReadLaneFirst | |
+
+## Wave Reductions
```

Comment:
It's weird to say that the two operations below are reductions but the ones above aren't, when many of the operations above are reductions.